libsolidity/ast/AsmJsonImporter.cpp

```diff
@@ -0,0 +1,276 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/**
+ * @author julius <djudju@protonmail.com>
+ * @date 2019
+ * Converts an inlineAssembly AST from JSON format to AsmData
+
+ */
+
+#include <libsolidity/ast/AsmJsonImporter.h>
+#include <libsolidity/ast/ASTJsonImporter.h>
```

Comment:
Duplicate include

Reply:
Look closer ;-).

Reply:
First time I saw it I myself actually thought "how can there be duplicate files here" :-).